kyuubi-server/web-ui/README.md

```diff
@@ -0,0 +1,39 @@
+# Kyuubi-ui
+
+### Start Using
+
+For the best experience, we recommend using `node 16.15.1` and `pnpm 7.9.0`.
+You can learn how to install the corresponding version from their official website.
+
+- [node](https://nodejs.org/en/)
+- [pnpm](https://pnpm.io/)
+
+### Install Dependencies
+
+```shell
+pnpm install
+```
+
+### Development Project
+
+To do this you can change the VITE_APP_DEV_WEB_URL parameter variable as the service url in .env.development in the project root directory, such as http://127.0. 0.1:8090
+
+
+```shell
+pnpm dev
+```
+
+### Build Project
+
+```shell
+pnpm run build
+```
+
+### Code Format
+
+Usually after you modify the code, you need to perform code formatting operations to ensure that the code in the project is the same style.
+
+```shell
+pnpm run prettier
```

Comment:
we provded a shell script `dev/reformat` for same purpose on backend codes. would you like to merge it into `dev/format` or provide another one at `kyuubi-server/web-ui/reformat` or in some place?